### PR TITLE
Mobile click scroll improve

### PR DESCRIPTION
**Pull request by jeffreyxuan**
1. Do no show x-direction scroll in mobile view
2. hard to click issue : iPhone does not like onClick without css cursor:pointer

### CODE REVIEW

**Comment by jeffreyxuan** [comment]
[(1) Mobile View - Scrolling not smooth [1]](https://trello.com/c/x8Gj7UmC/175-1-mobile-view-scrolling-not-smooth-1)

**Comment by jeffreyxuan** [comment]
[(1) Mobile View > Remind Me - Difficult to click [1]](https://trello.com/c/3q3CfSt8/167-1-mobile-view-remind-me-difficult-to-click-1)

File: client/src/modules/index/components/QuarterView/Mobile.js [hunk 1]
```diff
@@ -92,22 +92,21 @@ export default class Mobile extends Component {
...
-                  )}
                 </Label>
+                {serviceInfo.footnote && (
+                  <Footnote
+                    onClick={() => {
+                      onDayClick(formattedDate, serviceInfo);
+                    }}>
+                    {serviceInfo.footnote}
+                  </Footnote>
+                )}
```
Comment (josephj) [inline client/src/modules/index/components/QuarterView/Mobile.js]: Nice job man! It works so smooth. My solution was really foolish when it compares to yours.

File: client/src/modules/index/components/QuarterView/Mobile.js [hunk 2]
```diff
@@ -235,3 +239,8 @@ const Action = styled.span`
   margin-left: auto;
   margin-right: 20px;
 `;
+const BottomDateBarSpace = styled.div`
+  position: absolute;
+  height: 50px;
+  width: 100%;
+`;
```
Comment (josephj) [inline client/src/modules/index/components/QuarterView/Mobile.js]: Interesting. I don't know how this works. If it's absolute positioning, it shouldn't occupy any space. I tried and it also works when I remove the `position: absolute` and `width: 100%`.

Would it be easier if you specify the `margin-bottom: 50px` for the [last `<Row/>` element](https://github.com/efcsydney/efcsydney-roster/blob/master/client/src/modules/index/components/QuarterView/Mobile.js#L208) in Mobile View?

**Pull request merged**